js/data/line_bucket.js

```diff
@@ -377,6 +383,14 @@ LineBucket.prototype.addCurrentVertex = function(currentVertex, flip, distance,
     }
     this.e1 = this.e2;
     this.e2 = this.e3;
+
+    // There is a maximum "distance along the line" that we can store in the buffers.
+    // When we get close to the distance, reset it to zero and add the vertex again with
+    // a distance of zero.
```

Comment:
Please clarify that this maximum distance is determined by the number of bits in the `linesofar` attribute